Backend/Models/Messages/CommandMessage.cs

```diff
@@ -173,6 +179,56 @@ private void ProcessMute()
             SendAlert($"User has been Muted: {_target}. Reason: {_with}");
         }
 
+        private void ProcessUnmute()
+        {
+            if (_sender == null)
+            {
+                CLogger.Error("Command not invoked: Missing sender");
+                return;
+            }
+
+            if (UserManager.IsUserAdmin(_sender) == false)
+            {
+                CLogger.Error("User must be an adminstrator to use this command");
+                SendRefuse("You must be an adminstrator to use this command");
+                return;
+            }
+
+            if (_target == null)
+            {
+                CLogger.Error("Mute target not specified");
+                SendRefuse("Please indicate the user to unmute");
+                return;
+            }
+
+            User? userToUnmute = UserManager.GetUserByUsername(_target);
+            if (userToUnmute == null)
+            {
+                CLogger.Error("Unmute target not found in DB");
+                SendRefuse("User not found");
+                return;
+            }
+
+            if (!userToUnmute.IsMuted)
+            {
+                CLogger.Error("Command not invoked: User is not muted");
+                SendRefuse("User is not muted");
+                return;
+            }
+
+            if (userToUnmute == _sender)
+            {
+                CLogger.Error("Command not invoked: You cannot unmute yourself");
```

Comment:
The logs shouldn't be directed towards the user, they should simply state what happened, something like "Command not invoked: User tried to unmute themselves" or whatever